### ISSUE TITLE
failure of a unit test

### ISSUE THREAD

**Issue by maverickg** [stan-dev/rstan#195]
There is a failure in one of the tests outside rstan that won't be picked by R CMD check.

By the way, he test can be called by

```
$ R -q -f runRunitTests.R
```

where runRunitTests.R is at https://github.com/stan-dev/rstan/tree/develop/rstan/tests

The following is the part showing the error.  I figured out the problem, so I am gonna fix it. 

```
Test file: /home/runner/buildrstan/rstan/rstan/tests/unitTests/runit.test.stanfit.R 
test_grad_log: (5 checks) ... OK (28.76 seconds)
test_init_zero_exception_inf_grad: (2 checks) ... OK (28.48 seconds)
test_init_zero_exception_inf_lp: (2 checks) ... OK (28.18 seconds)
test_init_zero_gradient_failure: (2 checks) ... OK (28.6 seconds)
test_output_csv_and_extract: (19 checks) ... OK (29.07 seconds)
test_specify_args_and_data: ERROR !! 
Error in sp[, "n_divergent__"] : subscript out of bounds
--------------------------
```

**Comment by syclik** [comment]
I might have written that when refactoring some of the code. Feel free to
remove if it's no longer relevant.

**Comment by maverickg** [comment]
fixed via cb15b70489818d7d64e844149e0cefb923143505

@syclik it's not from your code
